src/main/java/org/sagebionetworks/bridge/spring/filters/RequestFilter.java

```diff
@@ -113,6 +141,16 @@ protected void setRequestContext(RequestContext context) {
         BridgeUtils.setRequestContext(context);
     }
 
+    protected Cookie makeSessionCookie(String sessionToken, int expireInSeconds) {
+        Cookie cookie = new Cookie(SESSION_TOKEN_HEADER, sessionToken);
+        cookie.setMaxAge(expireInSeconds);
+        cookie.setPath("/");
+        cookie.setDomain("localhost");
+        cookie.setHttpOnly(false);
+        cookie.setSecure(false);
+        return cookie;
+    }
```

Comment:
Similarly here. If we need this in two places, we should make a shared utility function.

Reply:
Hopefully, you don't need the code you've moved into these filters anymore in the controllers, and the controllers can be simplified.